src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1012,9 +1014,13 @@ protected File prepareTempDockerfile(File dockerfile) throws PluginExecutionExce
             debug("Creating temp Dockerfile...");
             tempDockerfile = File.createTempFile("tempDockerfile", "");
             debug("temp Dockerfile: " + tempDockerfile);
-            tempDockerfilePath = tempDockerfile.toPath();
-            // set the tempDockerfile to be deleted when the JVM exits
-            tempDockerfile.deleteOnExit();
+            tempDockerfilePath = tempDockerfile.toPath(); // save name to clean up later
+            if (keepTempDockerfile) {
+                info("Saving temporary Dockerfile named:"+tempDockerfilePath);
```

Comment:
Perhaps use terminology in line with the option name so it's easier to find.  e.g.:
```
info("Keeping temporary Dockerfile: " + tempDockerfilePath);
```

Reply:
Flashy! Done.